core/src/main/scala-3/anorm/PackageCompat.scala

```diff
@@ -0,0 +1,203 @@
+package anorm
+
+import java.sql.Connection
+
+import scala.util.Try
+
+private[anorm] trait PackageCompat:
+
+  extension (query: SqlQuery)
```

Comment:
More Scala2 implicit conversion that previously pimps `SqlQuery` as `SimpleSql[Row]` with extended combinators